BSP/Simulator/Src/BSP_Contactors.c

```diff
@@ -0,0 +1,48 @@
+#include "BSP_Contactors.h"
+#include <stdio.h>
```

Comment:
This was already included in common.h

Reply:
Removed for being redundant: #include <stdio.h>